econml/dml.py

```diff
@@ -605,8 +741,14 @@ def fit(self, Y, T, X=None, W=None, *, sample_weight=None, sample_var=None, grou
         """
         return super().fit(Y, T, X=X, W=W,
                            sample_weight=sample_weight, sample_var=sample_var, groups=groups,
+                           cache_values=cache_values, monte_carlo_iterations=monte_carlo_iterations,
                            inference=inference)
 
+    @DML.model_final.setter
+    def model_final(self, model):
+        raise AttributeError("LinearDML final model can't be chnaged from "
+                             "StatsModelsLinearRegression(fit_intercept=False)")
+
```

Comment:
We can't change final stage model but we could change some arguments under that model? e.g. cov_type, like what you did for `SubsampledHonestForest`

Reply:
Another typo here I just randomly saw it, "changed"

Reply:
Yes, I think that would make sense - but for now, cov_type is set based on the inference object's cov_type, it's never set directly by the user, so I just left that behavior as is for now.  Ultimately this ties back into our discussion of whether to change the DML hierarchy by unifying the LinearDML and SparseDML classes and cleaning up how we implement inference, but that's out of scope for now.